### ISSUE TITLE
add_slider_widget callback with custom scalar bar not updating mesh

### ISSUE THREAD

**Issue by bejota** [pyvista/pyvista#897]
Hello,
I'm implementing a threshold slider with a scalar bar. I'd like to scale and position both. I've tried two approaches. The second approach revealed a bug.

1. `add_mesh_threshold` method: works well and as expected except there is no control over the slider position/style

2. `add_mesh` plus `add_slider_widget`: provides access to slider position/style. However, when I pass both `position_x` and `position_y` to the `add_mesh` `scalar_bar_options` argument, the mesh no longer responds to the slider updates. I receive the message: `WARNING:root:Encountered issue in callback: 'values'`. (Either of the position properties work individually but not together.)

Comment out either or both of these scalar bar args and the slider functions properly.
```
scargs=dict(
    position_x=0.5,
    position_y=0.1
)
```

I appreciate your work on this project. Thank you.

-----

**Minimal Example**

```py
import numpy as np 
import pyvista as pv
# Volume rendering is not supported with Panel yet
pv.rcParams["use_panel"] = False
pv.rcParams["volume_mapper"] = 'gpu'

''' plot '''
values = np.linspace(0, 10, 1000).reshape((20, 5, 10))
grid = pv.UniformGrid()
grid.dimensions = np.array(values.shape) + 1
grid.origin = (100, 33, 55.6)  # The bottom left corner of the data set
grid.spacing = (1, 5, 2)  # These are the cell sizes along each axis
grid.cell_arrays["values"] = values.flatten(order="F")  # Flatten the array!

scargs=dict(
    position_x=0.5,
    position_y=0.1
)

name='foo'
cmin=values.min()
cmax=values.max()
p = pv.Plotter(notebook=False)
p.add_mesh(grid,show_edges=True, name=name, clim=[cmin,cmax], scalar_bar_args=scargs)
p.add_mesh(grid.outline(),show_edges=True) 

def func(value): 
    ''' callback function '''
    thresh = grid.threshold(value, scalars="values")
    if thresh.n_points < 1:
        p.remove_actor(name)
    else:
        p.add_mesh(thresh,show_edges=True, name=name, clim=[cmin,cmax])
    return

p.add_slider_widget(callback=func,rng=(cmin,cmax))
p.show()

```

**System Information:**

```txt
--------------------------------------------------------------------------------
  Date: Mon Sep 21 17:22:03 2020 Central Daylight Time

                OS : Windows
            CPU(s) : 8
           Machine : AMD64
      Architecture : 64bit
       Environment : Python
NVIDIA Corporation : GPU Vendor
Quadro P1000/PCIe/SSE2 : GPU Renderer
4.5.0 NVIDIA 443.18 : GPU Version

  Python 3.8.5 (default, Sep  3 2020, 21:29:08) [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.25.3
               vtk : 8.2.0
             numpy : 1.19.1
           imageio : 2.9.0
           appdirs : 1.4.3
            scooby : 0.5.6
            meshio : 4.1.1
        matplotlib : 3.3.1
             PyQt5 : 5.9.2
             scipy : 1.5.0

  Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------

```

**Comment by akaszynski** [comment]
Thanks for the MWE.  I found the error and we'll have this fixed in 0.26.1.  See PR #899

**Comment by akaszynski** [comment]
`pyvista==0.26.1` is live on pypi with the fix.
